urllib3/exceptions.py

```diff
@@ -196,6 +196,14 @@ class ResponseNotChunked(ProtocolError, ValueError):
     pass
 
 
+class BodyNotHttplibCompatible(AssertionError, ValueError):
```

Comment:
I'd like to see what @sigmavirus24's thinking was here: I'm wrong frequently enough that I like it when others confirm my thoughts. =D